static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js

```diff
@@ -0,0 +1,122 @@
+define([
+    'bean',
+    'qwery',
+    'lib/config',
+    'lib/detect',
+    'lib/cookies',
+    'commercial/modules/user-features'
+], function (
+    bean,
+    qwery,
+    config,
+    detect,
+    cookies,
+    userFeatures
+) {
+    return function () {
+        var self = this;
+        this.id = 'BundleDigitalSubPriceTest1';
+        this.start = '2017-04-24';
+        this.expiry = '2017-05-25'; // Thursday 25th May
+        this.author = 'Justin Pinner';
+        this.description = 'Test digital subs price points via thrasher';
+        this.showForSensitive = true;
+        this.audience = 0.25;
+        this.audienceOffset = 0;
+        this.successMeasure = '';
+        this.audienceCriteria = 'Non-paying UK network front users - tablet resolution and above';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'Find the price that works for most people.';
+        this.hypothesis = 'One of our price points will be more desirable than the others';
+
+        this.canRun = function () {
+            return document.querySelector('#membership-ab-thrasher') &&
+                    !cookies.getCookie('GU_DBPT1') &&
+                    !userFeatures.isPayingMember() &&
+                    detect.isBreakpoint({min: 'tablet'}) &&
+                    config.page.isFront &&
+                    config.page.pageId.toLowerCase() === "uk" &&
+                    config.page.edition.toUpperCase() === 'UK';
+        };
+
+        this.thrasherContainer = function() {
+            return document.querySelector('#membership-ab-thrasher');
```

Comment:
I'm tempted to leave this as-is right now on the basis that it's how it was last time around. If we decide to adjust the audience or applicable fronts prior to activating the test, I'll change it then.